frontend/packages/console-plugin-sdk/src/typings/notification-alerts.ts

```diff
@@ -0,0 +1,24 @@
+import { Extension } from './base';
+import { Alert } from '@console/internal/components/monitoring/types';
+
+namespace ExtensionProperties {
+  export interface AlertAction {
+    /* Alert name for the key in alertAction Map */
```

Comment:
I would add comment like this, which is tied to implementation. Better would be something like
```
/* Alert name as defined by `alert.rule.name` property */
```